### PR TITLE
Moves the AWS Scenario Guide out of the main docs

### PR DESCRIPTION
**Pull request by acozine**
##### SUMMARY

The Scenario Guides belong with the modules and other collection-based resources they describe. This PR removes the first one (AWS) from the core repo. https://github.com/ansible-community/antsibull/pull/255 allows publication of .rst files from collections to docs.ansible.com. https://github.com/ansible-collections/amazon.aws/pull/362 adds this content to the amazon.aws collection. 

We'll need to merge the other two PRs first, so we can uncomment and test the link to the AWS Scenario Guide in its new location. 

##### ISSUE TYPE
- Docs Pull Request

##### COMPONENT NAME
docs.ansible.com

### CODE REVIEW

**Comment by felixfontein** [comment]
Shouldn't we replace docs/docsite/rst/scenario_guides/guide_aws.rst by a stub which has a ref to `ansible_collections.amazon.aws.docsite.aws_intro`?

**Comment by acozine** [comment]
Ah, good point @felixfontein, I forgot about that.

**Review by felixfontein — APPROVED** [review]

**Comment by acozine** [comment]
rebuld_merge

**Comment by acozine** [comment]
Whoops, typo. Trying again to kick CI without merging.

/azp run

**Comment by felixfontein** [comment]
Needs a rebase due to #75375.

**Comment by felixfontein** [comment]
@samccann @ariordan-redhat now that amazon.aws 2.0.0 has been released with this guide (it's available at https://docs.ansible.com/ansible/devel/collections/amazon/aws/docsite/guide_aws.html), I think it's time to complete this PR!

**Review by tremble — APPROVED** [review]
LGTM

File: docs/docsite/rst/scenario_guides/cloud_guides.rst [hunk 1]
```diff
@@ -1,16 +1,19 @@
...
+**************************
+Legacy Public Cloud Guides
+**************************
 
-The guides in this section cover using Ansible with a range of public cloud platforms. They explore particular use cases in greater depth and provide a more "top-down" explanation of some basic features.
+The legacy guides in this section may be out of date. They cover using Ansible with a range of public cloud platforms. They explore particular use cases in greater depth and provide a more "top-down" explanation of some basic features.
+
+Guides for using public clouds are moving into collections. We are migrating these guides into collections. Please update your links for the following guides:
+
+:ref:`ansible_collections.amazon.aws.docsite.aws_intro`
```
Comment (tremble) [inline docs/docsite/rst/scenario_guides/cloud_guides.rst:11]: Actually, I'm failing to see this file in the Amazon.AWS collection

**Review by tremble — CHANGES_REQUESTED** [review]

File: docs/docsite/rst/scenario_guides/cloud_guides.rst (hunk 1 above)
Comment (tremble) [reply docs/docsite/rst/scenario_guides/cloud_guides.rst:11]: Am I looking in the right place?   https://github.com/ansible-collections/amazon.aws/tree/main/docs/docsite

Comment (felixfontein) [reply docs/docsite/rst/scenario_guides/cloud_guides.rst:11]: See the label: https://raw.githubusercontent.com/ansible-collections/amazon.aws/main/docs/docsite/rst/guide_aws.rst

Comment (ariordan-redhat) [reply docs/docsite/rst/scenario_guides/cloud_guides.rst:11]: @tremble
It's here: 
https://raw.githubusercontent.com/ansible-collections/amazon.aws/main/docs/docsite/rst/guide_aws.rst

Comment (tremble) [reply docs/docsite/rst/scenario_guides/cloud_guides.rst:11]: Ah the label right up at the top!

**Review by tremble — APPROVED** [review]

**Review by ariordan-redhat — APPROVED** [review]
Looks good to me

**Pull request merged**